en: update restart tidbcluster document (#58)
pingcap/docs-tidb-operator: en/restart-a-tidb-cluster.md

# Restart a TiDB Cluster in Kubernetes

If you find that the memory leak occurs in a Pod of the TiDB cluster during use, you need to restart the cluster. This document describes how to logoff a Pod and restart a TiDB cluster gracefully.

> **Warning:**
>
> It is not recommended to manually remove a Pod in the TiDB cluster without graceful restart in a production environment, because this might lead to some request failures of accessing the TiDB cluster though the `StatefulSet` controller pulls the Pod up again.

## Enable the configurations

> To activate the graceful logoff feature, you need to enable some Operator-related configurations. These configurations are disabled by default, follow the next step to manually turn them on.

Reviewer: ```suggestion
To activate the graceful logoff feature, you need to enable some related configurations in TiDB Operator. These configurations are disabled by default. Take the following steps to manually turn them on.
```

1. Edit the `values.yaml` file.

    Enable the `Operator Webhook` feature:

    ```yaml
    admissionWebhook:
      create: true
      mutation:

>         pods: true

Reviewer 2: These 2 lines should be removed.

Reviewer: @Yisaer Shall we remove them from the Chinese doc as well?

Reviewer 2: Yes, it has already been removed.

    ```

> 2. Install or update the TiDB Operator.

Reviewer: ```suggestion
2. Install or update TiDB Operator.
```

>     To install or update the TiDB Operator. see [Deploy TiDB Operator in Kubernetes](deploy-tidb-operator.md).

Reviewer: ```suggestion
    To install or update TiDB Operator, see [Deploy TiDB Operator in Kubernetes](deploy-tidb-operator.md).
```

> ## Use annotate to mark the targe Pod

Reviewer: ```suggestion
## Use annotate to mark the target Pod
```

> ## Use annotate to mark the targe Pod

Reviewer: ```suggestion
## Use annotate to mark the target Pod
```

> You can use `kubectl annotate` to mark the target Pod component of the TiDB cluster. After marking, the TiDB Operator automatically perform graceful logoff the Pod and restart the cluster. To mark the target Pod, run the following command:

Reviewer: ```suggestion
You can use `kubectl annotate` to mark the target Pod component of the TiDB cluster. After marking, the TiDB Operator automatically performs graceful logoff of the Pod and restarts the cluster. To mark the target Pod, run the following command:
```

> You can use `kubectl annotate` to mark the target Pod component of the TiDB cluster. After marking, the TiDB Operator automatically performs graceful logoff of the Pod and restarts the cluster. To mark the target Pod, run the following command:

Reviewer 2: I tend to use `restarts the target Pod` instead of `restarts the cluster`.
